x509/build/fmc_alias_cert_tbs.rs

```diff
@@ -0,0 +1,212 @@
+/*++
```

Comment:
You should add a presubmit test that ensures this code is exactly the same as the generated code, similar to what we do for the generated register code:

https://github.com/chipsalliance/caliptra-sw/blob/b72c29345cb3734b7263968af16f757a58d8a505/.github/workflows/build-test-verilator.yml#L100

Reply:
Done